packages/braid-design-system/src/lib/components/private/touchable/debugTouchable.ts

```diff
@@ -1,13 +1,12 @@
 import type { StyleRule } from '@vanilla-extract/css';
 
-export const debugTouchable = ({ after = false } = {}): StyleRule =>
-  process.env.NODE_ENV === 'production'
```

Comment:
Not sold on removing this check.

### Pro of stripping the style
- Any accidental debug attributes in code dont cause red boxes for our users

### Cons
- Cant debug in Playroom or Storybook without further work

Reply:
You could invert the condition and change the name of the envar to be something other than `NODE_ENV`, and only set it for playroom/storybook.
E.g. `process.env.DEBUG_TOUCHABLE ? {debugStyle} : {}`